index.html

```diff
@@ -0,0 +1,197 @@
+          West Blue, Ohara of all places. Put on your favorite cosplay costume,
+          because we will bazinga like it is our last day.
+        </p>
+        <p class="fs-1 fw-bold text-light">Mon 29 - Wen 31 May</p>
+        <p class="text-light fw-semibold">@Colosseum Ohara West Blue</p>
+      </div>
+    </header>
+    <main>
+      <section id="program" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 text-light">Main Program</h2>
+        <span class="line-primary"></span>
+        <div
+          class="d-flex flex-column flex-md-row flex-wrap justify-content-md-center m-3"
+        >
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-masks-theater fs-1 p-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Perfomances</h3>
+            <p class="p-1">
+              A lot of performers will join us at that time to blow our mind.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-pen-to-square fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Autographs</h3>
+            <p class="p-1">
+              All your favorite authors will gather to make your dream a
+              reality.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-handshake fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">
+              Cosplayer meet&greet
+            </h3>
+            <p class="p-1">
+              Every big cosplayers from all over the world will be there.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-sharp fa-solid fa-star fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Special events</h3>
+            <p class="p-1">
+              The "gear fourth" show will be the wild card but more special
+              events will also take place.
+            </p>
+          </div>
+        </div>
+        <button type="button"
+          class="btn btn-danger p-3 rounded-0 text-white fw-bold mx-auto text-uppercase"
+        >
+          Claim your badge
+        </button>
+      </section>
+      <section id="guest" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 mb-3">Our virtual guests</h2>
+        <span class="line-primary"></span>
+        <ul class="row mt-5 list-unstyled p-3 justify-content-md-between"></ul>
```

Comment:
The margins to the left and right of the speakers seems a bit small, meanwhile the spaces inbetween the left and right speakers is bigger as shown below. Kindly check and ensure that the dimensions (paddings, margins etc) is similar to the design. 

Your design

![Screenshot from 2023-01-12 08-37-34](https://user-images.githubusercontent.com/65233860/212018935-d0fea1df-6c06-4292-96ac-a42b6a896272.png)



Actual design

![Screenshot from 2023-01-12 08-37-41](https://user-images.githubusercontent.com/65233860/212018959-71ccd6d8-9845-40e2-aca9-68762139c946.png)